tests/test_ocr.py

```diff
@@ -154,7 +154,7 @@ def test_match_text_stringify_result():
 
     assert re.match(
         r"TextMatchResult\(timestamp=None, match=True, region=Region\(.*\), "
-        r"frame=1280x720x3, text=u'Onion Bhaji'\)", str(result))
+        r"frame=<1280x720x3>, text=u'Onion Bhaji'\)", str(result))
```

Comment:
I do like this angle bracket thing.  We will be able to add some more info when #364 lands too.